docs/source/reference/functions.rst

```diff
@@ -1,8 +1,8 @@
 Wrapper Functions
 =================
 
-Wrapper functions are differentiable (backward-able) functions, that take an array or a Chainer :class:`Variable` as an argument and return a Chainer :class:`Variable`, or a tuple of a Chainer :class:`Variable`.
-Wrapper functions may not have learnable parameters when used in models. This is to prevent the functions from changing output due to adjustment of learned parameters during training.
+Wrapper functions are backward-able functions (probably differentiable), that take an array or a Chainer :class:`Variable` as an argument and return a Chainer :class:`Variable`, or a tuple of a Chainer :class:`Variable`.
+Wrapper functions should not have learnable parameters when used in models and should not be members of the :class:`Chain`. This is to prevent the functions from changing output due to adjustment of learned parameters during training.
```

Comment:
It might be rare but wrapper functions can be members of a model as below.  I'm in favor of warning that adding a wrapper function to an instance of `Link` as a member has no special effect by Chainer.  No special effect doesn't mean it shouldn't.

An example is `activation` in `chainer/links/connection/mlp_convolution_2d.py`.  Similar example exists in ChainerRL, where a ready-to-use model class has a `self.nonlinearity` member in order to select activation `F.relu`, `F.tanh`, etc.